src/app/app.module.ts

```diff
@@ -9,6 +9,8 @@ import { FooterModule } from './common/components/footer/footer.module';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { HttpClientModule, HTTP_INTERCEPTORS } from '@angular/common/http';
 import { AuthInterceptor } from './common/auth-interceptor';
+import { LoaderModule } from './common/components/loader/loader.module';
+import { LoaderInterceptor } from './common/components/loader/loader.interceptopr';
```

Comment:
Minor. Typo in the file name? Should it be **interceptor** instead **interceptopr**?

Reply:
Fixed